pkg/utils/utils.go

```diff
@@ -59,6 +59,15 @@ var (
 	PublicStorageProviderID = "7993447f-687f-490d-875c-ac95e89a62a4"
 	// PublicStorageSpaceID is the space id used by the sharestorageprovider
 	PublicStorageSpaceID = "7993447f-687f-490d-875c-ac95e89a62a4"
+	// VirtualSpaceOwner is the non-existing user which is needed for public share impersonation
+	VirtualSpaceOwner = userpb.User{
+		Id: &userpb.UserId{
+			OpaqueId: "dbdf733d-427f-4ac3-8093-2cba5c307e4c",
```

Comment:
should we use something more speaking like `virtual-public-link-user-id`?

Reply:
Ok, we can do this safely because it is not a providerID?

I was just following the pattern for the provider IDs

Reply:
Mmmh. We should stick to uuids in case we add some validation later imo. Can we just move it to a `var VirtualPublinLinkUID`? Then we can use this everywhere in the code.

Reply:
you need an extra var? what about `utils.VirtualSpaceOwner.Id.GetOpaqueId()`

Reply:
Yes. Sorry. That's obviously good enough :+1: